### Laelaps laelaps_control/IsDescLoaded service
#
# Test if robot description has been loaded successfully.
#

---
bool is_desc_loaded   # specific robot model is [not] loaded and parsed
